### ISSUE TITLE
jest toMatchInlineSnapshot with webstorm issues

### ISSUE THREAD

**Issue by chrisblossom** [wallabyjs/public#2019]
### Issue description or question
Webstorm does not handle external file changes very well because you basically are forced to manually re-sync the file. This causes issues with wallaby when using Jest's [`toMatchInlineSnapshot`](https://jestjs.io/docs/en/snapshot-testing#inline-snapshots) feature and pressing `Update snapshot` inside the webstorm editor while the file is active.

If possible it would be very helpful for wallaby to force a file refresh after updating the file.

### Code editor or IDE name and version

```
WebStorm 2019.1 EAP
Build #WS-191.5849.24, built on February 27, 2019
```
### OS name and version

`macOS 10.14.3`

**Comment by chrisblossom** [comment]
I think this is more than just a feature request. It borderlines a `bug` because it makes using `toMatchInlineSnapshot` almost unusable with wallaby.

**Comment by ArtemGovorov** [comment]
@chrisblossom From the last time we have tried, the issue was with Webstorm (not handling external file changes), and wasn't related to Wallaby. When we launch Jest in watch mode without Wallaby, and try to use the feature (update snapshot), Webstorm reacts in the exact same way.

BTW, the same feature in VS Code in Wallaby works as expected.

**Comment by chrisblossom** [comment]
@ArtemGovorov That is probably the case (I haven't personally tried) if you are running jest's watch mode inside webstorm's integrated terminal, but it is not the case when using a terminal outside of webstorm (such as iTerm2) because webstorm syncs the file automatically once you focus webstorm.

Yes I know VS Code works, but IMO is an inferior editor in many ways.

**Comment by chrisblossom** [comment]
Also, when using jest's watch mode, if you do try to focus webstorm too soon and edit the file before jest has a chance to update it, webstorm lets you know the file has changed externally. With wallaby it either changes back without warning causing the test to fail again (over and over again for that matter), or completely error out saying the snapshot cannot be found.

**Comment by ArtemGovorov** [comment]
@chrisblossom Thanks for the feedback, we'll be looking into the issue next week.

**Comment by ArtemGovorov** [comment]
The feature is implemented and published in v1.0.125.
